style.css

```diff
@@ -1,36 +1,35 @@
-/* stylelint-disable comment-empty-line-before */
-/* stylelint-disable property-no-unknown */
-/* stylelint-disable rule-empty-line-before */
-/* stylelint-disable declaration-block-semicolon-newline-after */
-/* stylelint-disable comment-whitespace-inside */
-/* stylelint-disable indentation */
+/* stylelint-disable no-eol-whitespace */
+/* stylelint-disable no-descending-specificity */
+/* stylelint-disable no-duplicate-selectors */
```

Comment:
Kindly remove the commented codes. [Stylelint](https://stylelint.io/) is a linter for your stylesheets that helps you avoid errors and enforce conventions. please let's not try to bypass them

You can run `npx stylelint "**/*.{css,scss}" --fix` to automatically fix the stylelint errors in you CSS. 👽

If you will need an additional explanation or support please, kindly visit [this FAQ](https://microverse.zendesk.com/hc/en-us/articles/360026625973-How-can-I-get-help-from-TSEs-) to set up a 1:1 meeting with the Senior code reviewer. 😊